star_tides/core/actions/create_contact_action.py

```diff
@@ -0,0 +1,31 @@
+'''star_tides.core.action.create_contact_action
+'''
+
+from star_tides.services.databases.mongo.schemas.contact_schema import ContactSchema
+from star_tides.services.databases.mongo.models.contact_model import ContactModel
+from star_tides.core.data.contact_data import ContactData
+from star_tides.core.actions.base_action import Action
+
+
+class CreateContactAction(Action):
+    '''
+    CreateContactAction creates the given Contact in the database and
+    returns the saved copy.
+    '''
+
+    def __init__(self, contact: ContactData) -> None:
+        self.contact = contact
+        print(f'got contact! {contact}')
+
+    def run(self):
+        if self.contact.id is not None:
+            raise Exception('expected contact not to have ID')
+        # TODO(i/15): Add this back in once ContactSchema works with the
+        # update_timestamps field.
+        #
+        # self.contact.update_timestamps.append(
+        #     UpdateTimestampData(timestamp=datetime.now(),
+        #                         user_id=self.get_caller_user_id()))
+        schema = ContactSchema().load(self.contact._asdict())
+        saved_model = ContactModel(**schema, _created=True).save()
+        return ContactData.from_model(saved_model)
```

Comment:
You should be able to dump the model in the schema with something like
```
ContactSchema().dump(saved_model)
```
which should return a dictionary of the model.

Reply:
Acknowledged - I think I'll keep the ContactData here.